plugins/out_stackdriver/stackdriver.c

```diff
@@ -729,7 +731,41 @@ static int get_stream(msgpack_object_map map)
     return STREAM_UNKNOWN;
 }
 
-static int pack_json_payload(int operation_extracted, int operation_extra_size,
+static void validate_insert_id(msgpack_object * subobj, const msgpack_object * o, 
+                               insert_id_status * status)
+{
+    int i = 0;
+    msgpack_object_kv * p = NULL;
+    *status = INSERTID_NOT_PRESENT;
+
+    if (o == NULL || subobj == NULL) {
```

Comment:
Do we need to check if `subobj == NULL` ?

Reply:
Removed